docs/ticket-triage.rst

```diff
@@ -0,0 +1,26 @@
+=============
+Ticket Triage
+=============
+
+Ticket triage is a weekly, informal ritual that is meant to solve the problem of
+tickets getting opened and then forgotten about. It is simple and keeps project
+momentum going and prevents ticket cruft.
+
+It fosters conversation around project tasks and philosophies as they relate to 
+milestones.
+
+Process
+-------
+- The role of Ticket Triager rotates weekly 
+- The Triager needs admin status on ``Trac``
+- The Triager looks at all the tickets that have been created in the last week 
+    - They can use a custom query or do this as the week progresses 
+    - BONUS ROUND: Dig up a stale ticket from the past
+- Assign each ticket to a milestone on the Roadmap 
+- The following situations merit discussion:
+    - A ticket doesn't have an appropriate milestone and we should create one 
+    - A ticket, in vanishingly rare circumstances, should be deleted
+    - A ticket could be assigned to multiple milestones 
+    - There is another question about a ticket
+- These tickets will be brought to a weekly meeting (currently Tuesdays) for discussion 
+- Ticket Triager role is also assigned at this meeting 
```

Comment:
Aha.  That answers one of my questions above.  But we skipped that part today. ;)  Perhaps this information could be moved up.